docs/release_docs/AHA-Prod_ReleaseFlow.md

```diff
@@ -0,0 +1,36 @@
+# AHA-Prod Workflow
+Explaination of AHA-PROD workflow.
+![AHA-PROD](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/AHA-PROD_Workflow.png?raw=true)
+
+## AHA-PROD-ci-cd
+
+This workflow allows you to manually trigger a GitHub Action with a input Github ``` Tag_name ```, without having to push or create a pull request
+
+### JOBS
+
+#### Github-ECR-Tag-Check
+This is how Github-ECR-Tag-Check job looks like
+
+![github-ecr-tag-checks](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/Github-ECR-Tag-Check.png?raw=true)
```

Comment:
No need to add the code screenshots in the docs. Because the code may change over time like the versions of the actions being used. We should only explain the functionality here.